verl/trainer/config/ppo_trainer.yaml

```diff
@@ -802,6 +802,9 @@ trainer:
   # Logging backends to use: "console", "wandb", etc.
   logger: [ 'console', 'wandb' ]
 
```

Comment:
since verl integrates with multiple tracking backends, each may have different options. what about having this in the config instead:

```
logger_kwargs:
    entity: null
```
In the implementation, each backend only pass the kwargs to it's constructor if the value is not null.

Reply:
highly agree with @eric-haibin-lin and as `entity` is wandb specific config and please do not setup it in the `__init__` param list. 

```
logger_cfg = config.get('logger_kwargs', {})

proxy = logger_cfg.get("wandb_proxy", None)
entity = logger_cfg.get("entity",  None)
```